### PR TITLE
feature(decide): initial decide feature

### PR DESCRIPTION
**Pull request by asdqwex**

### CODE REVIEW

File: src/modules/decide.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+var random = require('random-js')()
+
+module.exports = {
+  names: ['decide'],
```
Comment (Kingles) [inline src/modules/decide.js]: maybe 'pick' as well?

File: src/modules/decide.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  onMessage: function (bot, user, userID, channelID, message, rawEvent) {
+    var messages = message.split(' ')
+    messages.shift()
+    messages.splice(1, 1)
```
Comment (Kingles) [inline src/modules/decide.js]: consider maybe using random-js' `pick` function to make this a little more concise. example below

``` js
  onMessage: function (bot, user, userID, channelID, request, rawEvent) {
    const message = random.pick(request.split(' ').slice(1).filter(x => x !== 'or'))
    bot.sendMessage({
      to: channelID,
      message
    })
  }
```

**Comment by Kingles** [comment]
Also tests would be good if you get the time! I can help out with setup if you'd like

File: src/modules/decide.js (hunk 1 above)
Comment (asdqwex) [inline src/modules/decide.js]: Agreed, added in.

File: src/modules/decide.js (hunk 2 above)
Comment (asdqwex) [inline src/modules/decide.js]: Reads much better so I have implemented.

**Pull request merged**